external/fv3fit/fv3fit/keras/_models/shared/halos.py

```diff
@@ -161,7 +161,7 @@ def append_halos(ds: xr.Dataset, n_halo: int) -> xr.Dataset:
 def append_halos_tensor(n_halo: int, tensor: tf.Tensor) -> tf.Tensor:
     """
     Given a tensor with
-    "tile", "x", "y", and "z" dimensions and no halo data,
+    "sample", "tile", "x", "y", and "z" dimensions and no halo data,
```

Comment:
Can you add a parameter for the name "sample" or maybe infer it automatically? Would be nice to apply e.g. with `time` as  dimension too.